build.gradle

```diff
@@ -50,6 +50,10 @@ tasks.withType(JavaCompile) {
       // com.stripe.param.AccountUpdateParams.Individual.Address] within this file.)
       // We should fix this by having autogen use the fully-qualified class to eliminate ambiguity.
       check("SameNameButDifferent", net.ltgt.gradle.errorprone.CheckSeverity.OFF)
+
+      // InlineMe (https://errorprone.info/docs/inlineme) seems neat, but in order to add these annotations
+      // we would be imposing another dependency on `errorprone` to our users, not worth it.
+      check("InlineMeSuggester", net.ltgt.gradle.errorprone.CheckSeverity.OFF)
```

Comment:
Why was this necessary?

Reply:
"errorprone" is our linter. We use it just in CI and don't impose errorprone as a dependency to our users. One of the linter errors in "errorprone" will take `@Deprecated` annotations that you add (part of standard Java), and if it determines that the implementation of the deprecated thing is simple enough to be inlined, it will suggest that you use an "@InlineMe" annotation instead (not part of standard Java, only in errorprone).

Makes sense if you're Google (the authors of errorprone) and you can impose a standard of errorprone being installed everywhere in your vast Java codebase, but if you're an open-source library that wants to make the library as dependency-free and easy to install as possible, the linter rule doesn't make sense.